system/Database/SQLSRV/Builder.php

```diff
@@ -290,6 +290,15 @@ private function getFullName(string $table): string
         }
 
         if ($this->db->escapeChar === '"') {
+            if (str_contains($table, '.')) {
+                $dbInfo    = explode('.', $table);
+                $database  = str_replace('"', '', $dbInfo[0]);
+                $schema    = str_replace('"', '', $dbInfo[1]);
+                $tableName = str_replace('"', '', $dbInfo[2]);
```

Comment:
probably validate first if `$dbInfo` index 1 and 2 exists first?

Reply:
Done.